src/app/public/modules/page-summary/page-summary-content.component.ts

```diff
@@ -1,5 +1,8 @@
 import { Component } from '@angular/core';
 
+/**
+ * Specifies content to display in the arbitrary section of the page sumamary.
```

Comment:
```suggestion
 * Displays content in the arbitrary section of the page summary.
```

Reply:
@Blackbaud-ToddRoberts, are you good with getting rid of the recommendation we have in place now: `We recommend that you display content to support the key tasks of users who visit the page. We also recommend that you keep in mind the context of how users will use the content and limit the content to avoid overloading the summary.`? Or should we port this over to the Design tab?